sysdump/sysdump.go

```diff
@@ -263,6 +267,14 @@ func NewCollector(k KubernetesClient, o Options, startTime time.Time, cliVersion
 			}
 			c.log("ℹ️  %s ConfigMap not found in %s namespace", ciliumConfigMapName, c.Options.CiliumNamespace)
 		}
+		if c.CiliumConfigMap != nil && len(c.CiliumPods) > 0 {
+			ciliumVersion, err := c.Client.GetCiliumVersion(context.Background(), c.CiliumPods[0])
```

Comment:
Yikes! hopefully we'll have a better way to retrieve version info... perhaps helm values??

Reply:
yeah this is not great, but it needs to work even when you install cilium without doing `helm install`, like `helm template | kubectl apply -` or similar.